parser/libflux_parser.go

```diff
@@ -6,11 +6,16 @@ import (
 	"encoding/json"
 
 	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/internal/parser"
 	"github.com/influxdata/flux/internal/token"
 	"github.com/influxdata/flux/libflux/go/libflux"
 )
 
 func parseFile(f *token.File, src []byte) (*ast.File, error) {
+	if !useRustParser() {
```

Comment:
Another question is, do we want to call `os. Getenv()` every time we parse a file?

Reply:
Are you concerned about the performance?  It didn't occur to me that it would be more than reading a global variable, but I guess it is, but a factor of 1000:
https://dinolai.com/notes/golang/golang-benchmark-reading-config-from-os-env-and-file.html

I'l open another PR to cache the value of the environment variable after the first time we read it.

Reply:
Yes, my concern is around the performance. I think this is on a critical path.